src/UI/Seller/src/app/account/components/account-summary/account-summary.component.ts

```diff
@@ -23,15 +24,17 @@ export class AccountSummaryComponent extends AccountContent {
     changeDetectorRef: ChangeDetectorRef,
     currentUserService: CurrentUserService,
     @Inject(applicationConfiguration) appConfig: AppConfig,
-    appAuthService: AppAuthService
+    appAuthService: AppAuthService,
+    middleware: MiddlewareAPIService
```

Comment:
Can you please remove the dependency on middleware service? Anything in there should get rolled into the headstart sdk which is now part of the project. Long term I'd like that service to be gone